rootex/core/renderer/shaders/custom_post_processing_vertex_shader.hlsl

```diff
@@ -0,0 +1,19 @@
+Texture2D<float4> Texture : register(t0);
+sampler Sampler : register(s0);
+
+struct VSInputTx
+{
+	float4 Position : SV_Position;
+	float2 TexCoord : TEXCOORD0;
+};
+
+VSInputTx main(uint vI : SV_VertexId)
+{
+	VSInputTx vout;
+
+	float2 texcoord = float2((vI << 1) & 2, vI & 2);
+	vout.TexCoord = texcoord;
+
+	vout.Position = float4(texcoord.x * 2 - 1, -texcoord.y * 2 + 1, 0, 1);
+	return vout;
+}
```

Comment:
look at out other vertex shaders and change the structure to them like consistent naming conventions. 
Why do we need a texture/sampler here?

Reply:
fixed